content/chronograf/v1.9/about_the_project/release-notes-changelog.md

```diff
@@ -8,7 +8,31 @@ menu:
     parent: About the project
 ---
 
+## v1.9.3 [2022-02-01]
+
+### Features
+- Add ability to rename TICKscripts.
+- Changes to the `Queries` page:
+  - Add `CSV download` button. 
+  - Rename `Running` column to `Duration`.
+  - Add `Status` column. When hovering over the `Duration` column, status shows `Kill` confirmation button.
+  - Modify the `CSV` export to include the `Status` column.
+- Upgrade to use new `google.golang protobuf` library.
+
+### Bug Fixes
+- Log remote URL when ping fails.
```

Comment:
I think needs more context.  This is a logging enhancement to provide additional information about which data source connection is having troubling connecting (based on the ping failure).

Reply:
Reworded:

- Ability to log the InfluxDB instance URL when a ping fails, making connection issues easier to identify.